README.md

```diff
@@ -1,7 +1,10 @@
 # DPNS Contract
 
-[![Build Status](https://img.shields.io/travis/com/dashevo/dpns-contract.svg?branch=master&style=flat-square)](https://travis-ci.com/dashevo/dpns-contract)
-[![NPM version](https://img.shields.io/npm/v/@dashevo/dpns-contract.svg?style=flat-square)](https://npmjs.org/package/@dashevo/dpns-contract)
+[![NPM Version](https://img.shields.io/npm/v/@dashevo/dpns-contract)](https://www.npmjs.com/package/@dashevo/dpns-contract)
+[![Build Status](https://travis-ci.com/dashevo/dpns-contract.svg?branch=master)](https://travis-ci.com/dashevo/dpns-contract)
+[![Release Date](https://img.shields.io/github/release-date/dashevo/dpns-contract)](https://github.com/dashevo/dpns-contract/releases/latest)
+[![license](https://img.shields.io/github/license/dashevo/dpns-contract.svg)](LICENSE)
```

Comment:
I think we can skip the license badge. Github displays it pretty prominently in the sidebar and it is also at the bottom of the readme.
```suggestion
```